packages/adapters/subgraph/.graphclientrc.yml

```diff
@@ -326,9 +329,9 @@ sources:
           - endpoint: "https://gateway-arbitrum.network.thegraph.com/api/{env.GRAPH_API_KEY}/subgraphs/id/6oJrPk9YJEU9rWU4DAizjZdALSccxe5ZahBsTtFaGksU"
             retry: 2
             timeout: 30000 # 30 seconds
-          - endpoint: "https://api.thegraph.com/subgraphs/name/connext/amarok-runtime-v1-gnosis"
-            retry: 5
-            timeout: 30000 # 30 seconds
+          # - endpoint: "https://api.goldsky.com/api/public/project_clssc64y57n5r010yeoly05up/subgraphs/connext/amarok-runtime-v1-gnosis/gn"
+          #   retry: 5
+          #   timeout: 30000 # 30 seconds
```

Comment:
do we want the goldsky endpoint commented out?

Reply:
Yes, since it ran into indexing errors, and we need to check with their team